apps/web/pages/_app.tsx

```diff
@@ -44,10 +43,6 @@ function MyApp(props: AppProps) {
       <Head>
         <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
       </Head>
-      <Script
-        id="calcom-version"
-        dangerouslySetInnerHTML={{ __html: `window.CalComVersion = '${CalComVersion}'` }}
-      />
       {getLayout(
```

Comment:
@hariombalhara i'm removing this until we need it

Reply:
Cool